src/library_pthread_stub.js

```diff
@@ -4,6 +4,13 @@
  * SPDX-License-Identifier: MIT
  */
 
+#if USE_PTHREADS
+#error Internal error! USE_PTHREADS should not be enabled when including library_pthread_stub.js.
+#endif
+#if STANDALONE_WASM && SHARED_MEMORY
+#error STANDALONE_WASM does not support shared memories yet
+#endif
```

Comment:
Does it make sense to move this `STANDALONE_WASM` check to here?  I seems rather odd here.

Reply:
This check is paired in library_pthread.js for the pthreads case, and here for the non-pthreads case. To me it felt symmetric to have it in these files. It would be odd to do this check in library_pthread.js for the pthreads case, and in emcc.py for the non-pthreads case.

Could leave this in emcc.py, but slimming emcc.py felt like an improvement here.